### DISCUSSION TITLE
v0 Beta

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#4446] [Show & Tell]
### What is v0?

[v0](https://v0.dev/) is a generative UI system built by [Vercel](https://vercel.com/)'s AI team. It produces [React](https://react.dev/) code based on [shadcn/ui](https://ui.shadcn.com/) and [Tailwind CSS](https://tailwindcss.com/). It’s currently in Beta and you can join the waitlist now!

### Where should I go for help?

The Vercel AI team set up v0 Discord where you can share ideas, get help, ask for feature changes, and submit bug reports. If you're off the waitlist, check your invite email for the link.

### Where should I go for more information?

You can read more about v0 at https://vercel.com/blog/announcing-v0-generative-ui

**Comment by AM-young-fun** [comment]
Heyheyhey, I'm eager to have a alpha test qulification. Can you grant me that?

**Reply by amyegan** [reply]
People are currently being added from the waitlist. Please sign up on [v0.dev](https://v0.dev) if you haven't already 😄

**Reply by AM-young-fun** [reply]
Thanks! I've got the access to it, and I think it's really coool!!

**Comment by XadillaX** [comment]
Hi, does v0 has any public roadmap?

**Reply by amyegan** [reply]
There are no specific dates yet, but you can find a list of future features at [vercel.com/blog/announcing-v0-generative-ui#what's-ahead](https://vercel.com/blog/announcing-v0-generative-ui#what's-ahead)

**Comment by Mahir-Solution** [comment]
how to resolve this issue         This Serverless Function has crashed.

Your connection is working correctly.

Vercel is working correctly.

500: INTERNAL_SERVER_ERROR
Code: FUNCTION_INVOCATION_FAILED
ID: bom1::f89wg-1699341495229-b5e090d8a13d

**Reply by amyegan** [reply]
Please [start a new discussion](https://github.com/vercel/community/discussions/new) with more details about your situation 🙏

**Comment by leeanddrew** [comment]
How long does it usually take to grant access from the waitlist? Thank you!

**Reply by zederbear** [reply]
I'm wondering this as well

**Comment by ccarse** [comment]
Super excited for v0. Any idea when more invites will be going out? Thanks!

**Comment by CarlosZiegler** [comment]
Hey, I am writing to express my keen interest in participating in the alpha testing of your upcoming Version 0 platforme :) I am really excited about that :)

**Comment by harsh-ps-2003** [comment]
I am a freelancer and I would really appreciate if private beta access could be given to me. I am waitlisted from months... @amyegan

**Comment by amyegan** [comment]
Thanks for reaching out about v0 access! There has been a huge amount of interest in this. Invites are being sent out in batches, but I don't have an exact time estimate. 

The team is working quickly to increase system capacity and update features with [Private Beta](https://vercel.com/docs/release-phases#private-beta) feedback. More people will get access soon. 😄 

https://v0.dev/faq

**Comment by donttouchmylancable** [comment]
Can i get access?
I already bought the **basic plan**, but i can't do anything with it.

**Reply by amyegan** [reply]
Hi @donttouchmylancable. I want to make sure I understand the situation correctly. Are you saying that you were able to buy the basic plan without having access to v0?

**Reply by amyegan** [reply]
Thanks for reporting this! We identified the bug and pushed a solution so no one else ends up in this situation. In your case, we've gone ahead and sent you an invite so you can actually use your subscription. Please let me know if that works for you 🙂

**Comment by MokumJ** [comment]
How does this waitlist work? I entered the waitlist a couple of hours after first announcement and still don't got an invite. My colleague who entered a couple of days later is already in for over a month? Seems weird...

**Reply by CarlosZiegler** [reply]
Same here, friend from me get access right now but I am waiting :(

**Comment by donttouchmylancable** [comment]
Thank you really much <3

Amy ***@***.***> schrieb am Di., 28. Nov. 2023, 00:28:

**Comment by amyegan** [comment]
No more waitlist! v0 is now open for everyone 🎉 

https://x.com/vercel/status/1735719381739454730

**Comment by l4b4r4b4b4** [comment]
is v0 open source licensed?

**Reply by amyegan** [reply]
You can find more info about v0 in these docs:
- [Pre-release Agreement](https://v0.dev/agreement)
- [Vercel AI Policy](https://v0.dev/policy)
- [Frequently Asked Questions](https://v0.dev/faq)

**Reply by l4b4r4b4b4** [reply]
Dont understand the referenced material... Its a simply yes or no question, to you the maintainer. And by the looks of it the answer seemt to be no, right?

**Reply by amyegan** [reply]
If you're asking if v0 is open source, the answer is no.

If you're asking how it can be used, the answer is more complicated and can be found in the linked docs. 😄

[a comment by an automated account is not shown]
